### PR TITLE
Class6/yohannes

### PR DESCRIPTION
**Pull request by byohannes**
I have done a simple website for a youth educational charity using semantic HTML.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -1,19 +1,73 @@
...
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
- */
+.nav .nav-links {
+  float: right;
```
Comment (LucyMac) [inline css/style.css]: Please don't use floats, they cause layout issues and have been replaced by flexbox which is much easier and more predictable to use. 
Can you edit this to use flexbox instead?

File: index.html [hunk 2]
```diff
@@ -1,21 +1,79 @@
...
+          <li>Who WE Are</li>
+          <li>What We Do</li>
+          <li>Our Movement</li>
+          <li>Get Involved</li>
+        </ul>
+        <img class="e-logo" src="img/logo.png" alt="Logo Image" width="" />
+      </div>
+    </nav>
+    <section class="section-one">
+      <div class="sec1"></div>
```
Comment (LucyMac) [inline index.html]: You're duplicating markup unnecessarily here. `sec1` doesn't do anything - you can remove it.

File: index.html [hunk 3]
```diff
@@ -1,21 +1,79 @@
...
+    </section>
+    <section class="section-two">
+      <div class="sec2">
```
Comment (LucyMac) [inline index.html]: Again, why 2 elements here, when only one would be enough?

File: css/style.css [hunk 4]
```diff
@@ -1,19 +1,73 @@
...
+  margin-top: 7.875rem;
+  padding: none;
+  color: #1d1e21;
+  text-align: center;
+}
+.section-two .buttons {
+  text-align: center;
+  margin-top: 1.563rem;
+}
+.lowe {
```
Comment (LucyMac) [inline css/style.css]: Make sure you name things using easy-to-understand words. I'm not sure what `lowe` means.

File: css/style.css [hunk 5]
```diff
@@ -1,19 +1,73 @@
...
 
+.section-two .sec2 {
+  margin-top: 7.875rem;
+  padding: none;
+  color: #1d1e21;
+  text-align: center;
+}
+.section-two .buttons {
+  text-align: center;
+  margin-top: 1.563rem;
```
Comment (LucyMac) [inline css/style.css]: Can you add styles to your button so it looks nicer?

**Comment by byohannes** [comment]
Thanks, For the feedback Lucy. I am working to fix on all the mistakes and also to include media queries so the updated version of the homework will be up as a PR.

**Pull request closed without merging**